### PR TITLE
New Species: Diona

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR 
Adds Diona, the newest round-start species of loveable plant people.

Diona are a slow-moving species that generally just lumber along. Mechanically, they have fire vulnerability, high thirst, and slow speed. Ideally, they'd need light to survive, but that's a bit out of initial scope. They also have a limited number of organs, similar to slime, and are unable to fit shoes on their root-like feet.

Otherwise, they're just like any other crew member.
**Screenshots**

image

**Changelog**

:cl:
- add: Added the Diona species. This is a group of slow-moving plant people who just want to help out. Be kind to them on the station and keep them away from fires.

### CODE REVIEW

**Comment by Emisse** [comment]
looks cool

**Comment by Froozigiusz** [comment]
What would be their name list look like? Some plantoid name-list from Stellaris, Unfallen from ES2 or there is already a name-list from SS13? What name structure would they have?

**Comment by theashtronaut** [comment]
They should have very flow-y flower-y names. Stuff like "Flowers-of-Dew" "Fronds-of-Purple"

File: Resources/Prototypes/Entities/Mobs/Species/diona.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      types:
+        Bloodloss:
+          1
+    bloodlossHealDamage:
+      types:
+        Bloodloss:
+          -0.25
+  - type: Inventory
+    templateId: diona
+  - type: MovementSpeedModifier
```
Comment (Elijahrane) [inline Resources/Prototypes/Entities/Mobs/Species/diona.yml:39]: Are they the same mass (70 kg) as other species? If not you can adjust their physics component

File: Resources/Prototypes/Species/diona.yml [hunk 2]
```diff
@@ -0,0 +1,118 @@
+- type: species
+  id: Diona
+  name: species-name-diona
+  roundStart: true
+  prototype: MobDiona
+  sprites: MobDionaSprites
+  markingLimits: MobDionaMarkingLimits
+  dollPrototype: MobDionaDummy
+  skinColoration: TintedHues
+
```
Comment (Elijahrane) [inline Resources/Prototypes/Species/diona.yml:10]: Is there a nameset floating around somewhere you could use? Maybe check whichever BYOND codebase you ported them from?

File: Resources/Prototypes/InventoryTemplates/diona_inventory_template.yml [hunk 3]
```diff
@@ -0,0 +1,104 @@
+﻿#human but no shoes
```
Comment (Elijahrane) [inline Resources/Prototypes/InventoryTemplates/diona_inventory_template.yml:1]: If they're supposed to be made of wood maybe you should give them the "ShoesRequiredStepTriggerImmune" tag so they aren't griefed by broken glass

File: Resources/Prototypes/Body/Mechanisms/diona.yml [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    size: 1
+    compatibility: Biological
+  - type: SolutionContainerManager
+    solutions:
+      stomach:
+        maxVol: 250
+  - type: Stomach
+  - type: Metabolizer
+    maxReagents: 6
+    metabolizerTypes: [ Human ]
```
Comment (Elijahrane) [inline Resources/Prototypes/Body/Mechanisms/diona.yml:91]: They should probably get a plant metabolism so they have unique interactions with botany reagents

**Comment by asperger-sind** [comment]
> What would be their name list look like? Some plantoid name-list from Stellaris, Unfallen from ES2 or there is already a name-list from SS13? What name structure would they have?

Bonus points if you name yourself the Baol and get burnt down.

**Comment by EmoGarbage404** [comment]
recolored markings, added random name gen, added some QOL tweaks rane suggested, and added special effects for botanical reagents.

**Comment by mirrorcult** [comment]
"not needing to breathe air" is really lame as we saw with slimes already and it doesnt make any sense thematically

**Review by Elijahrane — APPROVED** [review]

**Comment by Elijahrane** [comment]
> "not needing to breathe air" is really lame as we saw with slimes already and it doesnt make any sense thematically
Can figure something out with CO2 maybe. It's a bit challenging because station mix doesn't have any trace CO2 by default, while IRL there's about 0.04% trace CO2. At very least, they should be okay in normal station air but not in a vaccuum.

**Comment by EmoGarbage404** [comment]
On the note about slimes, i feel that a lot of the issues with slimes is just that they had no need for air with nothing else that serves as a downside. This meant that they basically were just humans with negligible downsides and one big upside. On the other hand, Diona, while not needing air, are slow-moving, weak to fire, and cannot wears shoes (such as magboots, which hurts their ability in space). I would think that all of that would generally be enough to dissuade the type of powergaming players which typically used slimes in the old fashion.

**Comment by ZeroDayDaemon** [comment]
Making them breath carbon dioxide would be cool and make sense tho

**Comment by ZeroDayDaemon** [comment]
and they exhale oxygen

**Comment by mirrorcult** [comment]
the issues with slimes not needing air is that it ended up having shitty knock on effects (not suffocating in crit) and meant people could just ignore a game mechanic entirely, rather than just having to think about it differently and make different choices which is more interesting

**Comment by EmoGarbage404** [comment]
i think breathing co2 would be cool but i don't think there's any co2 in standard mix, which doesn't make it very viable for breathing. I suppose i could just make them breathe O2 like everything else; it's not a feature i'm willing to die on.

**Comment by ZeroDayDaemon** [comment]
Adding a small amount of carbon dioxide to standard mix for them to be able to breath but not enough to poison humans etc, since plants need alot less carbon dioxide than humans need oxygen would work?

**Comment by Elijahrane** [comment]
If you are planning photosynthesis in the far future I think having O2 breathing as a placeholder until then would be OK, when they get that it won't be 'ignoring a game mechanic' so much as 'having an alternate game mechanic that accomplishes the same thing'

**Comment by EmoGarbage404** [comment]
alright, they breath air now

**Pull request merged**